main/tasks/power_management_task.c

```diff
@@ -97,6 +99,10 @@ void POWER_MANAGEMENT_task(void * pvParameters)
 
     vTaskDelay(500 / portTICK_PERIOD_MS);
     uint16_t last_core_voltage = 0.0;
+    
+    uint16_t last_known_asic_volt = 0;
+    uint16_t last_known_asic_freq = 0;
```

Comment:
The `uint16_t last_known_asic_freq` is not needed, it's only written to NVS for backwards compatibility, in case someone downgrades to pre 2.10.0 (#1051). In code you can let the float be leading, just make sure you also write the int version if needed.

It might be neater to pull this backwards compatibility code into `nvs_config`, but that's for a different PR.